### DISCUSSION TITLE
vc-drawings/vc-measurements : measureUnits, customize labels

### DISCUSSION THREAD

**Discussion by PetukhovArt** [zouyaoji/vue-cesium#595] [Q&A]
Hi ! 
Some questions with vc-measurements : 
1) show label (name of graphics) in the center of graphics (polygon,circle , ..etc) 
2) point label , how to customize showing text values? 
i can't understand , how to await cesiumObject is ready and primitives are set , this only works with manual setTimeout
and if i draw new graphics , labels reset to default text
![image](https://github.com/user-attachments/assets/1e8802e9-7d21-4584-acd9-1482958c2801)

3) dynamic change data of drawing options such as show labels ( i dont want to show labels in different conditions , may be height)

**Comment by zouyaoji** [comment]
You could consider obtaining the drawn data, such as drawingsRef.value.getDrawingActionInstances()[2].cmpRef.value.renderDatas.value, and then convert it to CZML.

**Reply by PetukhovArt** [reply]
i dont understand why i need convert to it and how to combine it with vc-measurements... (
 
now on every drawEvt i save actual renderData (also add custom name and id)  into json and sync it with server, after component mount set renderData from this json 

can i dynamically manipulate labels with my approach ? 

- show/unshow name of primitive
- show/unshow full label or it items ( length/square/angle/slope .. etc)
-  change color of primitive
- change name of primitive

if, to use full control i need to use czml, can you link some example or describe how to use czml with vc-measurements
